src/path.js

```diff
@@ -23,12 +23,29 @@ function appendRound(digits) {
   };
 }
 
+function equal(x0, y0, x1, y1) {
+  return x0 === x1 && y0 === y1;
+}
+
+function equalRound(digits) {
+  const d = Math.floor(digits);
+  if (d > 15) return equal;
```

Comment:
```suggestion
  if (!(d <= 15)) return equal;
```

Use pessimistic check in case of NaN.